trunk/SUAVE/Components/Energy/Cooling/Cryocooler.py

```diff
@@ -135,10 +145,11 @@ def energy_calc(self, cooling_power, cryo_temp, amb_temp):
 
             print("Warning: The required cryogenic temperature of " + str(cryo_temp) + " is not achievable using a " + self.cooler_type + " cryocooler at an ambient temperature of " + str(amb_temp) + ". The minimum temperature achievable is " + str(temp_min))
 
-        self.mass_properties.mass     = mass
-        self.rated_power              = input_power
+        self.outputs.mass_properties.mass     = mass
```

Comment:
Just curious, is this a variable mass or a fixed property?

Reply:
I believe it's a variable mass since it depends on the maximum cooling power required of the cryocooler .

Reply:
I went back and reread the comments that are made above. It should not be a variable. It should be set once during sizing and then not again when run normally.

If the mass_properties are not in the typical data structure, when the mass of the total vehicle is summed the mass won't be added.

Reply:
Okay that makes sense, I'll fix that up